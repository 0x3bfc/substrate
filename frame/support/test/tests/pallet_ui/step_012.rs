#[frame_support::pallet(Example)]
mod pallet {
	#[pallet::trait_]
	pub trait Trait: frame_system::Trait {}

	#[pallet::module]
	pub struct Module<T> {}
}

fn main() {
}
