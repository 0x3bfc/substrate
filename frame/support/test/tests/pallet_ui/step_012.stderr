error: Missing `#[pallet::interface]`
 --> $DIR/step_012.rs:2:1
  |
2 | mod pallet {
  | ^^^
